tokens/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Maybe I'm being inconsistent, but do we want this to be public or not?

I don't mind it, but I'm pondering: Not (even remotely) every change to the vue-components will result in a change to the tokens. Would be weird to publish no-diff changes in the tokens every time. Should we use [`--independent`](https://github.com/lerna/lerna/tree/master/commands/init#--independent) then? Or am I missing something?

Reply:
As far as I know they would get published together, even with no-diff changes. I'm in favor of just making this private for now until there is a use case.

Reply:
So after some team discussion we concluded that our existing apps already want/need a more elegant way to get to token than the build script that exists copied over and over. Publishing the tokens right away is the way to go then.
We looked a vue-cli and how it manages its sub-packages and concluded that it is also not absurd to increment all version when publishing a change to only one sub-package and that it is a nice touch to be able to verify all packages are in sync by seeing the same version number in all of them (in consuming package.jsons).